### PR TITLE
[10.0] mig account multicurrency revaluation report

### PR DESCRIPTION
**Pull request by yvaucher**
Depends on
- [x] #47

### CODE REVIEW

File: account_multicurrency_revaluation/demo/account_demo.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    <record id="reval_journal" model="account.journal">
+        <field name="name">Revaluation Journal</field>
+        <field name="code">RJ</field>
+        <field name="type">general</field>
+    </record>
+    <record id="bank_journal_usd" model="account.journal">
+        <field name="name">Bank Journal USD</field>
+        <field name="code">BJ USD</field>
+        <field name="type">bank</field>
+        <field name="currency_id" ref"base.USD"/>
```
Comment (grindtildeath) [inline account_multicurrency_revaluation/demo/account_demo.xml]: Please rebase on grindtildeath:10.0-mig-account_multicurrency_revaluation to correct ref=

File: account_multicurrency_revaluation/demo/account_demo.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <field name="default_debit_account_id" ref="demo_acc_liquidity_usd"/>
+        <field name="default_credit_account_id" ref="demo_acc_liquidity_usd"/>
+    </record>
+    <record id="bank_journal_gbp" model="account.journal">
+        <field name="name">Bank Journal GBP</field>
+        <field name="code">BJ GBP</field>
+        <field name="type">bank</field>
+        <field name="currency_id" ref"base.GBP"/>
```
Comment (grindtildeath) [inline account_multicurrency_revaluation/demo/account_demo.xml]: Please rebase on grindtildeath:10.0-mig-account_multicurrency_revaluation to correct ref=

**Review by grindtildeath — CHANGES_REQUESTED** [review]

**Comment by grindtildeath** [comment]
@yvaucher Please rebase on 10.0 and cherry-pick these three commits :
00a6548fe3a95351cb16a4172ef0f25629aa2a0b
b63c05ee2e71eba7d5703729001ad04117ecc6c0
006923d61dc556a98c5ea5b0e43ff58044757d83

**Review by grindtildeath — APPROVED** [review]

File: account_multicurrency_revaluation_report/README.rst [hunk 3]
```diff
@@ -12,7 +12,7 @@ Installation
 
 To install this module, you need to:
 
-* clone the branch 8.0 of the repository https://github.com/OCA/account-closing
+* clone the branch 9.0 of the repository https://github.com/OCA/account-closing
```
Comment (gurneyalex) [inline account_multicurrency_revaluation_report/README.rst]: change branch name

File: account_multicurrency_revaluation_report/README.rst [hunk 4]
```diff
@@ -40,7 +40,7 @@ Known issues / Roadmap
 ======================
 
 * The module depends on the module *base headers webkit* which is in work in
-progress in this pull request: https://github.com/OCA/webkit-tools/pull/10
+  progress in this pull request: https://github.com/OCA/webkit-tools/pull/10
```
Comment (gurneyalex) [inline account_multicurrency_revaluation_report/README.rst]: are you sure ? this is a PR for 8.0 and it's closed.  are we still using webkit report??

File: account_multicurrency_revaluation_report/__manifest__.py [hunk 5]
```diff
@@ -1,38 +1,21 @@
...
- "version": "8.0.1.0.0",
+ "version": "10.0.1.0.0",
  "category": "Finance",
  "author": "Camptocamp,Odoo Community Association (OCA)",
  "license": 'AGPL-3',
  "summary": "Module for printing reports that completes the module "
             "Multicurrency Revaluation",
  "depends": [
      "account_multicurrency_revaluation",
-     "base_headers_webkit"
```
Comment (gurneyalex) [inline account_multicurrency_revaluation_report/__manifest__.py]: well the README needs updating...

File: account_multicurrency_revaluation_report/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (gurneyalex) [inline account_multicurrency_revaluation_report/__openerp__.py]: should be __manifest__.py

File: account_multicurrency_revaluation_report/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright 2012-2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+{"name": "Multicurrency Revaluation Report",
+ "version": "9.0.1.0.0",
```
Comment (gurneyalex) [inline account_multicurrency_revaluation_report/__openerp__.py]: wrong version nuymber

**Review by gurneyalex — CHANGES_REQUESTED** [review]

**Comment by grindtildeath** [comment]
@gurneyalex I think the PR should be good now. I can't increase the coverage anymore because it's mostly the python class used for the report which is decreasing it, and IMO less than 2% should be ok.

File: account_multicurrency_revaluation/tests/test_currency_revaluation_report.py [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+from odoo.tests.common import TransactionCase
+
+
+class TestCurrencyRevaluationReport(TransactionCase):
+
+    def test_wizard_empty_accounts(self):
+        wizard = self.env['unrealized.report.printer']
+        wiz = wizard.create({})
+        data = {'lang': 'en_US',
+                'params': {'action': 254},
```
Comment (guewen) [inline account_multicurrency_revaluation/tests/test_currency_revaluation_report.py]: Does the action's id matter? The hardcoded number looks suspicious to me :)

**Review by gurneyalex — APPROVED** [review]

**Comment by yvaucher** [comment]
I'm the original author of this PR but this was reworked by @grindtildeath so I will make an exception to the rule, don't merge your own PR as I letted in WIP and is not finallized by me.

**Pull request merged**